web/src/components/AssertionForm/Fields/dropDownRenderComponent.tsx

```diff
@@ -0,0 +1,33 @@
+import {Tag} from 'antd';
+import parse from 'html-react-parser';
+import MarkdownIt from 'markdown-it';
+import React, {useMemo} from 'react';
+
+import {OtelReference} from '../hooks/useGetOTELSemanticConvertionAttributesInfo';
+import {referencePicker} from './referencePicker';
+
+const md = MarkdownIt();
+
+export function dropDownRenderComponent(
+  hoveredKey: string | undefined,
+  reference: OtelReference
+): (menu: React.ReactElement) => React.ReactElement {
+  return menu => {
+    const otelReferenceModel = referencePicker(reference, hoveredKey || '');
+    const description = useMemo(() => parse(md.render(otelReferenceModel.description)), [otelReferenceModel]);
+    return (
+      <div style={{display: 'inline-flex', width: '100%'}}>
```

Comment:
Let's prefer using styled-components instead of inline styles.

Reply:
ok